src/status_im2/contexts/contacts/events.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im2.contexts.contacts.events
+  (:require
+   [utils.re-frame :as rf]
+   [status-im.contact.db :as contact.db]
+   [status-im.data-store.contacts :as contacts-store]))
+
+(rf/defn load-contacts
+  {:events [:contacts/contacts-loaded]}
+  [{:keys [db] :as cofx} all-contacts]
+  (let [contacts-list (map #(vector (:public-key %)
+                                    (if (empty? (:address %))
+                                      (dissoc % :address)
+                                      %))
+                           all-contacts)
+        contacts      (into {} contacts-list)]
+    {:db (cond-> (-> db
+                     (update :contacts/contacts #(merge contacts %))
+                     (assoc :contacts/blocked (contact.db/get-blocked-contacts all-contacts))))}))
+
+(rf/defn initialize-contacts
+  [cofx]
+  (contacts-store/fetch-contacts-rpc cofx #(rf/dispatch [:contacts/contacts-loaded %])))
```

Comment:
Please, update your editor to follow the Clojure Style Guide to keep a newline character at the end of files.

Reply:
Not related to the issue but: Why was not fixed by `make lint-fix`? Should an issue for that be created?

Reply:
Unfortunately it might not be that easy. We use `zprint` to format the code and we couldn't find a way to specify that trailing newlines should be added to files. `zprint` is still evolving, just recently there was a new release with some bug fixes (we should probably upgrade soon). At least one of them due to a report from @yqrashawn.

Because we're trying to follow the majority of the Clojure Style Guide, my suggestion here would be to add a second step in `lint-fix` to add trailing newlines to all files reported by `find_all_clojure_files` in the `Makefile`. But as usual, it's also better that contributors configure their editors to rely less on `make lint-fix` so they can skip this extra step. I never need to call `make lint-fix`, although it's obviously an important make target to have for other reasons.